deploy.sh

```diff
@@ -40,6 +40,10 @@ if [ -f "$CONFIG_FILE" ]; then
   cp  "$CONFIG_FILE"  "$driver_dir/$config_file_name"
 fi
 
+echo https://$KUBERNETES_SERVICE_HOST:$KUBERNETES_SERVICE_PORT > $driver_dir/master_url
```

Comment:
Is this script shared between both the master and worker DaemonSets? If so we'll want to make sure we don't output this stuff in the worker DaemonSet.